tests/page-objects/common/common.wdio.page.js

```diff
@@ -222,6 +228,16 @@ const openAppManagement = async () => {
   await (await $('.navbar-brand')).waitForDisplayed();
 };
 
+const getDefaultLanguages = async () => {
```

Comment:
I think this is a misleading name, along with hardcoding values used in a specific test in common page objects. 
I think this function should return the languages only, and all assertions should belong in the test file.